### PR TITLE
Covscan fixes

### PR DESCRIPTION
**Pull request by evgenyz**
Some fixes for covscan of 1.3.2

### CODE REVIEW

**Comment by jan-cerny** [comment]
@evgenyz Looks great, but please rebase it and remove the "Merge" commit.

File: utils/oscap-chroot [hunk 1]
```diff
@@ -74,26 +81,23 @@ function usage()
...
-    echo "Invalid arguments provided."
-    usage
-    die
+    invalid "Invalid arguments provided."
 fi
 
 # Learn more at https://www.redhat.com/archives/open-scap-list/2013-July/msg00000.html
 export OSCAP_PROBE_ROOT
-OSCAP_PROBE_ROOT="$(cd "$1"; pwd)"
+OSCAP_PROBE_ROOT="$(cd "$1" && pwd)"
```
Comment (matejak) [inline utils/oscap-chroot]: This doesn't really handle the case when one can't cd into that directory - in that case, the `OSCAP_PROBE_ROOT` will be empty, which is not handled. I would go for calling `die` right afterwards if `OSCAP_PROBE_ROOT` is empty.

File: utils/oscap-podman [hunk 2]
```diff
@@ -92,14 +93,16 @@ for VAR in `podman inspect $ID --format '{{join .Config.Env " "}}'`; do
     eval "export OSCAP_OFFLINE_$VAR"
 done
 
-export OSCAP_PROBE_ROOT="$(cd "$DIR"; pwd)"
+export OSCAP_PROBE_ROOT
+OSCAP_PROBE_ROOT="$(cd "$DIR" && pwd)"
```
Comment (matejak) [inline utils/oscap-podman]: ditto

File: utils/oscap-vm [hunk 3]
```diff
@@ -128,7 +131,7 @@ fi
 
 # Learn more at https://www.redhat.com/archives/open-scap-list/2013-July/msg00000.html
 export OSCAP_PROBE_ROOT
-OSCAP_PROBE_ROOT="$(cd "$MOUNTPOINT"; pwd)"
+OSCAP_PROBE_ROOT="$(cd "$MOUNTPOINT" && pwd)"
```
Comment (matejak) [inline utils/oscap-vm]: ditto

File: utils/oscap-chroot (hunk 1 above)
Comment (evgenyz) [reply utils/oscap-chroot]: Added fall-back `die`.

File: utils/oscap-podman (hunk 2 above)
Comment (evgenyz) [reply utils/oscap-podman]: Situation here is not really like it is in `oscap-chroot`, but added as well, for sake of completeness.

File: utils/oscap-vm (hunk 3 above)
Comment (evgenyz) [reply utils/oscap-vm]: Situation here is not really like it is in `oscap-chroot`, but added as well, for sake of completeness.

**Comment by evgenyz** [comment]
Rebased and updated.

**Comment by matejak** [comment]
Thanks, merging!

**Pull request merged**